concrete/views/panels/details/page/composer.php

```diff
@@ -125,6 +125,24 @@
             }).submit();
         });
 
+        $('button[data-page-type-composer-form-btn=save]').on('click', function() {
+            my.disableAutosave();
+            var submitSuccess = false;
+            my.$form.concreteAjaxForm({
+                url: '<?=$controller->action('save')?>',
+                success: function(r) {
+                    submitSuccess = true;
+                    window.location.href = r.redirectURL;
+                },
+                complete: function() {
+                    if (!submitSuccess) {
+                        my.enableAutosave();
```

Comment:
Is this meant to be the error handler? Can we display an error here if access is denied or something else happens?